robotframework-ls/vscode-client/src/run.ts

```diff
@@ -53,6 +53,10 @@ async function checkFileExists(file) {
 }
 
 async function readAllDebugConfigs(workspaceFolder: WorkspaceFolder): Promise<DebugConfiguration[]> {
+    const launch = workspace.getConfiguration("launch");
+    const configs = launch.inspect<DebugConfiguration[]>("configurations").globalValue;
```

Comment:
It seems that `launch.inspect<DebugConfiguration[]>("configurations")` could return `undefined`, so, maybe this should be:

`launch.inspect<DebugConfiguration[]>("configurations")?.globalValue`

Reply:
You are right. VS Code did not warn me, because `strict` is not set in `vscode-client/tsconfig.json`. I think it should be set to true to prevent running into runtime errors.

Reply:
Agreed. I created https://github.com/robocorp/robotframework-lsp/issues/523 to track that.